genie-web/src/integTest/resources/application-integration.yml

```diff
@@ -29,6 +29,9 @@ genie:
     cluster-load-balancer:
       source: file:///tmp/genie/loadBalancers/script/source/loadBalance.js
       auto-load-enabled: true
+    execution-mode-filter:
+      source: file:///tmp/genie/loadBalancers/script/source/executionModeFilter.js
```

Comment:
when we change the name of `ClusterLoadBalancer` to `ClusterSelector` we probably want to change these paths too.

Reply:
Ah yeah, actually not even sure why these are there given that these files won't exist. Test that the app starts and work regardless?